odoo/orm/fields_properties.py

```diff
@@ -946,6 +949,9 @@ def _validate_properties_definition(self, properties_definition, env):
             if property_type and property_type not in Properties.ALLOWED_TYPES:
                 raise ValueError(f'Wrong property type {property_type!r}.')
 
+            if property_type == 'html' and (default := property_definition.get('default')):
+                property_definition['default'] = html_sanitize(default)
```

Comment:
Not sure but it would be good to recheck the parameters of the `html_sanitize`
`field.Html.sanitize_attributes is True` by default, but `html_sanitize`'s default parameter is `sanitize_attributes=False`